interface-definitions/service_config-sync.xml.in

```diff
@@ -5,6 +5,7 @@
       <node name="config-sync" owner="${vyos_conf_scripts_dir}/service_config-sync.py">
         <properties>
           <help>Configuration synchronization</help>
+          <priority>1</priority>
```

Comment:
use please 1000+ priority here
It should be triggered after all other nodes are configured
It checks if something is changed in a configured node section/subsection and then applies this if changed.

Reply:
Done